docs/source/upgrade-guide/index.md

```diff
@@ -30,6 +30,93 @@ the latest changes, and propose to you to merge the changes, so you can run it o
+The previous text block `text` based on `draftJS` is now deprecated and is restricted (hidden) in Volto bootstrap.
+This is a major change and should be planned in advance before you install 16.0.0-alpha.15 or later.
+
+```{versionadded} 16.0.0-alpha.15
+`volto-slate` added to Volto core as the default text block.
+```
+
+```{deprecated} 16.0.0-alpha.15
+`text` text block based on `draftJS` is now deprecated and might be removed from core in a future major version.
+```
+
+```{note}
+Volto 16 is intended to be the long-term support (LTS) version used when Plone 6 Final is released.
+Volto 16 will ensure compatibility with all existing Volto deployments for versions XXX and won't introduce any hard breaking change.
+As such you will have the choice to continue using `draftJS` text block in your projects.
+See below for more information.
+```
+
+These are the possible scenarios:
+
+- New projects
+- Existing projects, already using `volto-slate` add-on
+- Existing projects using core `draftJS`, opting to continue using `draftJS`
+- Existing projects using core `draftJS`, opting to start using `slate` without migrating (possible, but not recommended)
+- Existing projects using core `draftJS`, opting to migrate to `slate`
+
+#### New projects
+
+New projects are unaffected, since they will use Slate as the default text block from the beginning.
+
+#### Existing projects, already using `volto-slate` add-on
+
+For projects already using `volto-slate`, take the following steps in your project configuration:
+
+- Remove `volto-slate` add-on from your project build dependencies.
+- Add these lines to the Jest configuration in your project's `package.json`:
+
+```diff
+--- a/package.json
++++ b/package.json
+@@ -57,6 +57,7 @@
+       "^.+\\.js(x)?$": "babel-jest",
+       "^.+\\.css$": "jest-css-modules",
+       "^.+\\.scss$": "jest-css-modules",
++      "^.+\\.less$": "jest-css-modules",
+       "^.+\\.(png)$": "jest-file",
+       "^.+\\.(jpg)$": "jest-file",
+       "^.+\\.(svg)$": "./node_modules/@plone/volto/jest-svgsystem-transform.js"
+@@ -67,6 +68,7 @@
+     "moduleNameMapper": {
+       "@plone/volto/babel": "<rootDir>/node_modules/@plone/volto/babel",
+       "@plone/volto/(.*)$": "<rootDir>/node_modules/@plone/volto/src/$1",
++      "@plone/volto-slate": "<rootDir>/node_modules/@plone/volto/packages/volto-slate/src",
+```
+
+- If any of your code depends on `volto-slate` code, update your imports by adding the namespace `@plone/` to the original `volto-slate` import.
+
+```diff
+- import { DetachedTextBlockEditor } from 'volto-slate/blocks/Text/DetachedTextBlockEditor';
++ import { DetachedTextBlockEditor } from '@plone/volto-slate/blocks/Text/DetachedTextBlockEditor';
+```
+
+### Existing projects using core `draftJS`, opting to continue using `draftJS`
+
+You will have to configure your project to continue using `draftJS`, for example, in your `config.js` or in your add-on:
+
+```js
+config.settings.defaultBlockType = 'slate'
```

Comment:
@sneridagh This should be `config.settings.defaultBlockType = 'text'` if they want to continue using draftjs ?

Reply:
Good catch! Fixed!